.quickstart/quickstart.yml

```diff
@@ -20,6 +20,10 @@ table_variables:
     - audit_log
   using_holidays:
     - schedule_holiday
+  using_brands:
+    - brand
+  using_organizations:
+    - organization
```

Comment:
Believe we also need to add `organization_tag` here

Reply:
Good callout. However, the `using_organizations` variable being set to true doesn't require the `organization_tag` table to be enabled.  But we could argue that `using_organization_tags` requires `organization` to be added. What do you think?

Reply:
Ah yes thanks for catching, I had it flipped. I think it makes sense for `organization` to be added under `using_organization_tags` .